src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/benchmarks.wcfcore.Perf.yml

```diff
@@ -1,83 +1,60 @@
 jobs:
-  BasicHttpWithBuffered:
+  WCFCorePerf:
     source:
       repository: https://github.com/dotnet/wcf.git
       branchOrCommit: master
       project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
+    readyStateText: WCFCorePerf Client
     isConsoleApp: true
     waitForExit: true
-    variables:      
-      Url: http://wcfcranksrv01.redmond.corp.microsoft.com/SayHello/SayHello.svc
-      Binding: BasicHttp
-      Duration: 300
-      TransferMode: Buffered
-    arguments: " binding:{{Binding}} serviceurl:{{Url}} transfermode:{{TransferMode}} perfmeasurementduration:{{Duration}} "
-  BasicHttpWitDefault:
-    source:
-      repository: https://github.com/dotnet/wcf.git
-      branchOrCommit: master
-      project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
-    isConsoleApp: true
-    waitForExit: true
-    variables:      
-      Url: http://wcfcranksrv01.redmond.corp.microsoft.com/SayHello/SayHello.svc
-      Binding: BasicHttp
-      Duration: 300
-      TransferMode: Default
-    arguments: " binding:{{Binding}} serviceurl:{{Url}} transfermode:{{TransferMode}} perfmeasurementduration:{{Duration}} "
-  WSHttp:
-    source:
-      repository: https://github.com/dotnet/wcf.git
-      branchOrCommit: master
-      project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
-    isConsoleApp: true
-    waitForExit: true
-    variables:      
-      Url: https://wcfcranksrv01.redmond.corp.microsoft.com/wcfservicewshttp/service.svc
-      Binding: WSHttp
-      Duration: 300
-    arguments: " binding:{{Binding}} serviceurl:{{Url}}  perfmeasurementduration:{{Duration}} "
-  NetTcp:
-    source:
-      repository: https://github.com/dotnet/wcf.git
-      branchOrCommit: master
-      project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
-    isConsoleApp: true
-    waitForExit: true
-    variables:       
-      Url: net.tcp://wcfcranksrv01.redmond.corp.microsoft.com/WcfNetTcpService/Service1.svc
-      Binding: NetTcp
+    variables:
       Duration: 300
-    arguments: " binding:{{Binding}} serviceurl:{{Url}} perfmeasurementduration:{{Duration}} "  
+    arguments: " binding:{{Binding}} serviceurl:{{Protocol}}://{{Server}}{{Url}} transfermode:{{TransferMode}} perfmeasurementduration:{{Duration}} "
 
 scenarios:
   BasicHttpWithBuffered:    
     load:
-      job: BasicHttpWithBuffered
+      job: WCFCorePerf
       variables:
+        Server: /
+        Protocol: Http
+        Url: /SayHello/SayHello.svc
+        Binding: BasicHttp
+        TransferMode: Buffered
         serverPort: 8231
```

Comment:
Server ports should be set in the profiles, not in the scenario. Or if not in profiles, at least be a default value in the job if it's a common value.

Reply:
The ‘serverPort: 8231’ in the scenarios is not used, so I removed it. [cb04d79](https://github.com/dotnet/wcf/pull/4394/commits/cb04d796e5cf53e0f94211fb2370382039678ffc)